lib/Nodes/Table/TableColumn.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\RST\Nodes\Table;
+
+use Doctrine\RST\Nodes\Node;
+use LogicException;
+use function strlen;
+use function trim;
+use function utf8_encode;
+
+class TableColumn
```

Comment:
This, along with `TableRow` help `TableNode` work with meaningful objects, instead of a ton of arrays.

Reply:
Make this class final if you can.